### DISCUSSION TITLE
response-transformer rename for jsonbody

### DISCUSSION THREAD

**Discussion by zhongweikang** [Kong/kong#12824] [Help] [answered]
Why doesn't response-transformer implement rename for jsonbody, I see the corresponding implementation in request-transformer, which is theoretically possible

**Comment by chronolaw** [comment, marked as answer]
We will appreciate if you create a PR for this feature.

**Reply by zhongweikang** [reply]
Hey @chronolaw, can you help me with this? https://github.com/Kong/kong/pull/12888
Thank you so much.
